src/style.css

```diff
@@ -1,3 +1,77 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+header {
+  background-color: #fff;
+  flex-direction: row;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0;
+  padding-right: 50px;
+}
+
+nav {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 60px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  justify-content: center;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  display: flex;
+  height: 15px;
+  color: #000;
+  font-size: 12px;
+  font-weight: 500;
+  text-decoration: none;
+  text-transform: uppercase;
+  cursor: pointer;
+  margin: 0 10px;
+  padding: 22.5px 0;
+}
+
+.nav__link:hover {
+  color: #00acdc;
```

Comment:
this color can be stored in variable, and use here as, e.g.

`--main-color: #00acdc;`

`color: var(--main-color);`

Reply:
Move all repeatable colors to the CSS variables and use them.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)